### PR TITLE
Add some improvements

### PR DESCRIPTION
**Pull request by DumitruBogdan**
With this PR I want to:
- add another section where you can see the grand total of the check;
- change the tip percentage mechanism with a picker similar to the number of people picker;
- add a header for the last section, so it can be identifiable as the amount for person text area.

### CODE REVIEW

File: Shared/ContentView.swift [hunk 1]
```diff
@@ -10,19 +10,19 @@ import SwiftUI
...
     @State private var numberOfPeople = 0
-    @State private var tipPercentage = 10
+    @State private var tipPercentage = 0
     @FocusState private var amountIsFocused: Bool
     
     var tipPercentages = [0, 10, 15, 20, 25]
-    
-    var totalPerPerson: Double {
-        let peopleCount = Double(numberOfPeople + 2)
+    var checkTotal: Double{
```
Comment (vladstanescu94) [inline Shared/ContentView.swift]: Let's leave a space between Double and {

File: Shared/ContentView.swift [hunk 2]
```diff
@@ -35,23 +35,31 @@ struct ContentView: View {
                         .focused($amountIsFocused)
                     
                     Picker("Number of people", selection: $numberOfPeople) {
-                        ForEach(2 ..< 100) {
+                        ForEach(2 ..< 101) {
```
Comment (vladstanescu94) [inline Shared/ContentView.swift]: Maybe we should remove these magic numbers inside the ForEach

File: Shared/ContentView.swift [hunk 3]
```diff
@@ -35,23 +35,31 @@ struct ContentView: View {
...
                             Text("\($0) people")
                         }
                     }
+                } header: {
+                    Text("The check amount is: ")
                 }
                 Section {
                     Picker("Tip percentage", selection: $tipPercentage) {
-                        ForEach(tipPercentages, id: \.self) {
+                        ForEach(2 ..< 100) {
```
Comment (vladstanescu94) [inline Shared/ContentView.swift]: Same as the above

**Review by vladstanescu94 — CHANGES_REQUESTED** [review]
Looks good 👍, just some minor changes.

File: Shared/ContentView.swift (hunk 3 above)
Comment (DumitruBogdan) [reply Shared/ContentView.swift]: Resolved

File: Shared/ContentView.swift (hunk 2 above)
Comment (DumitruBogdan) [reply Shared/ContentView.swift]: Resolved

File: Shared/ContentView.swift (hunk 1 above)
Comment (DumitruBogdan) [reply Shared/ContentView.swift]: Resolved

**Pull request merged**